travis.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+build_cpp() {
+  ./autogen.sh
+  ./configure
+  make -j2
+  make distcheck -j2
```

Comment:
Doesn't distcheck take a while? I was thinking of distcheck as being a separate config. It's fine for the conformance test to be here, it's pretty fast (though it's not enabled for C++ yet).